### DISCUSSION TITLE
Demo description not showing up

### DISCUSSION THREAD

**Discussion by ISchwandner** [IBAHCM/RPiR#470] [Archive] [answered]
Hi all,

I keep having this issue that when I call demo(package = "Mypackage") on my packages that the description of the last demo in the list does not show up in the file that opens. I checked that the spacing between demo name and description is 3 spaces and it is exactly the same as in all other demos, where the description does show up. 

Does it have something to do with what happens at the end of the description? I have hit enter, should I type something specific or nothing at all?

Has anyone had this issue to and was able to resolve it?

Many thanks,

Imo

**Comment by soniamitchell** [comment, marked as answer]
Always check for typos.. 👍

**Comment by richardreeve** [comment]
@ISchwandner - in case you still haven't spotted the problem, in the `00Index` file, you have "D02_A-generic_diversity_function", whereas the filename is "D02_A_generic_diveristy_function" - there are two typos there!

**Comment by ISchwandner** [comment]
Many thanks Sonia and Richard!
